law-widget-styles/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@lawsafrica/law-widget-styles",
+  "version": "1.0.0",
+  "description": "Stylesheets for supporting AkomaNtoso HTML documents.",
+  "main": "",
+  "files": [
+    "/css",
+    "/scss"
+  ],
+  "scripts": {
+    "test": "jest",
+    "build": "sass --style compressed scss/law-widget-styles.scss css/law-widget-styles.min.css && sass scss/law-widget-styles.scss css/law-widget-styles.css"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/laws-africa/law-widget-styles.git"
+  },
+  "author": "Greg Kempe <greg@laws.africa>",
+  "license": "MIT",
```

Comment:
Change to match the law-widgets license (LGPL 3.0)

Reply:
done

Reply:
done